lib/sles4sap_publiccloud.pm

```diff
@@ -830,6 +836,13 @@ sub create_playbook_section_list {
         push @playbook_list, 'fully-patch-system.yaml';
     }
 
+    # Add playbook to download and install PTFs from azure, if any
+    if ($args{ptf_files} && $args{token}) {
+        $args{token} =~ s/\\\&/\&/g;
+        push @playbook_list, "ptf_installation.yaml -e ptf_files=$args{ptf_files} -e sas_token=$args{token} -e container=$args{container} -e storage=" . get_required_var('HANA_ACCOUNT');
```

Comment:
No need to change but personally I do not like that most of the input come as function arguments and then we read jut one  `HANA_ACCOUNT` from setting